nimare/io.py

```diff
@@ -137,12 +139,18 @@ def convert_sleuth_to_dict(text_file):
 
     # Split into experiments
     data = data[1:]
-    metadata_idx = [i for i, line in enumerate(data) if line.startswith('//')]
-    exp_idx = np.split(metadata_idx,
-                       np.where(np.diff(metadata_idx) != 1)[0] + 1)
-    start_idx = [tup[0] for tup in exp_idx]
+    exp_idx = []
+    for i in np.arange(0, len(data)):
+        if data[i].startswith('//') and data[i + 1].startswith('// Subjects') and data[i - 1].startswith('//') is False:
+            exp_idx.append(i)
+        elif data[i].startswith('//') and i + 2 < len(data) and data[i + 2].startswith('// Subjects'):
```

Comment:
This allows 1-2 line contrast names, but not 3+, right? Shouldn't be a problem the vast majority of the time, but I want to make sure.

Reply:
I believe that's correct! I can add another `if` to make allow for 3+ line contrast names, if you want?